DEPS

```diff
@@ -31,7 +31,7 @@ vars = {
   # Dart is: https://github.com/dart-lang/sdk/blob/master/DEPS.
   # You can use //tools/dart/create_updated_flutter_deps.py to produce
   # updated revision list of existing dependencies.
-  'dart_revision': '0cc70c4a7c448078dd2aea8e45c56bc50fbbb31c',
+  'dart_revision': 'cff0e751a59871190c4b8d8ee1a8604cb402933a',
```

Comment:
Can you instead roll to https://github.com/dart-lang/sdk/commit/011676641a8b4b77bb372384c712709cbf037675? The Dart SDK prior to that commit (including cff0e751a59871190c4b8d8ee1a8604cb402933a) is know to break Flutter, so I think it's safe to skip cff0e751a59871190c4b8d8ee1a8604cb402933a altogether.

Reply:
Done